com.unity.uiwidgets/Runtime/gestures/converter.cs

```diff
@@ -207,8 +207,11 @@ public static class PointerEventConverter {
                                 keyBoardEvent.command = (datum.modifier & (1 << (int) FunctionKey.command)) > 0;
                                 keyBoardEvent.control = (datum.modifier & (1 << (int) FunctionKey.control)) > 0;
 #endif
-                                RawKeyboard.instance._handleKeyEvent(keyBoardEvent);
-                                TextInput.OnGUI();
+                                //if there is a text input field attached then the input should be swallowed by it. Otherwise we pass the input to RawKeyboard
+                                if (!TextInput.OnGUI()) {
```

Comment:
Skip all key event during text edit, make direction key and delete key do not work during editing.

Reply:
fixed